src/Microsoft.AspNetCore.Server.IISIntegration/IISMiddleware.cs

```diff
@@ -18,6 +18,8 @@ namespace Microsoft.AspNetCore.Server.IISIntegration
 {
     public class IISMiddleware
     {
+        public static readonly string AuthenticationScheme = "Windows";
```

Comment:
Do you want to follow the pattern from Security? E.g. IISDefaults.AuthenticationScheme? 
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.Google/GoogleDefaults.cs#L11
Note those are public const string so they can be used in attributes.